### PR TITLE
Add collaborators names

### PR DESCRIPTION
**Pull request by kbariotis**
For #1113, this will bring back a previously-reverted PR and also add an input on the Room Dialog to be able to set your own name.

### CODE REVIEW

**Comment by Fausto95** [comment]
Two pointers here:
* We should ask the user's name before starting/joining a collaboration session.
* When we reset the canvas, we loose the username, should be fixed.

**Comment by dwelle** [comment]
@Fausto95 we discussed this on chat: we don't want to show prompts which would add friction to our app.

**Comment by lipis** [comment]
Let's put a darker border around the name, as we do it with the arrow.

File: src/renderer/renderScene.ts [hunk 1]
```diff
@@ -326,6 +327,30 @@ export function renderScene(
...
+      const measureHeight =
+        measure.actualBoundingBoxDescent + measure.actualBoundingBoxAscent;
+
+      context.fillRect(
+        offsetX,
+        offsetY,
+        measure.width + 2 * paddingHorizontal,
+        measureHeight + 2 * paddingVertical,
+      );
+      context.fillStyle = "white";
```
Comment (lipis) [inline src/renderer/renderScene.ts:346]:
```suggestion
      context.fillStyle = "#fff";
```

**Comment by lipis** [comment]
Screenshot 2020-04-07 at 10 51 57 AM

The background is too light.. make it the same as the arrow!

**Comment by kbariotis** [comment]
image

Paddings are correct @lipis if you count from the lower end of the word.

**Comment by lipis** [comment]
In your preview.. the background is still lighter than the arrow.. is that an old one? can you post the screenshot of the final form?

**Comment by kbariotis** [comment]
image

sorry! @lipis

**Comment by lipis** [comment]
We can improve the UX for the username later..

**Pull request merged**